### DISCUSSION TITLE
Facepalm emoji defaulting to man

### DISCUSSION THREAD

**Discussion by rupalivohra** [community/community#44004] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Hi GitHub,
I love that we can use emojis directly in text on PRs using the colon (:) syntax.
One of my frequently-used emojis is `:facepalm:`. On GitHub, however, `:facepalm:` yields 🤦🏽 . That's a man facepalming.
There's also `:man_facepalming:`: 🤦🏽‍♂️ and `:woman_facepalming:`: 🤦🏽‍♀️ , but that means that because I am a woman, I have to arrow down twice for accurate representation.

I know I'm the minority on here as a woman who is a software engineer, and that generally the easiest thing to do is to optimize for the majority. But I humbly request that you do better. Specifically, my request is that you make the default icon for `:facepalm:` gender neutral (e.g. maybe a smiley face rather than an actual person).

Thanks!
-Rupali

**Comment by daira** [comment]
All of the shortcuts for emoji that have a gender-neutral presentation (which I believe is now *all* of the potentially gendered ones, as Emoji 12.0 / 2019, if not earlier), should have the shortcut without "man_" or "woman_" point to the gender-neutral one.
